cmd/bad-key-revoker/main.go

```diff
@@ -76,6 +82,19 @@ func (ubk uncheckedBlockedKey) String() string {
 		ubk.RevokedBy, ubk.KeyHash)
 }
 
+func (bkr *badKeyRevoker) countUncheckedKeys() (int, error) {
+	var count int
+	err := bkr.dbMap.SelectOne(
+		&count,
+		`SELECT COUNT(*)
+		FROM (SELECT 1 FROM blockedKeys
+		WHERE extantCertificatesChecked = false
+		LIMIT ?) AS a`,
```

Comment:
Just for my own edification, why is this better/more efficient/more correct than
```
SELECT COUNT(*) FROM blockedKeys
WHERE extantCertificatesChecked = false
LIMIT 1000
```
? For example, we use this format [here](https://github.com/letsencrypt/boulder/blob/main/sa/sa.go#L2125).

Reply:
Yeah! So, the difference between these two queries is the `LIMIT` statement at the end of the one in this PR. Since `SELECT COUNT(*)` can only return one value, the `LIMIT 1` wouldn't actually be taken into account while counting the query. So instead, we break the selection of rows which should be limited out into its own sub-query and then return a `COUNT()` of the resulting rows.

You can experiment with this yourself by using your query above with just, replace the `LIMIT 1000` with a `LIMIT 1`, and then re-running the unit tests. The unit tests will all pass, even though the second case which calls this code will return with a row `COUNT()` of `2`.